deploy/mainnet/11_deploy_v1.0.1.ts

```diff
@@ -0,0 +1,42 @@
+import { DeployFunction } from "hardhat-deploy/dist/types";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+import { isDeployed, logStep, logStepEnd } from "../../ts-utils/helpers/index";
+
+const func: DeployFunction = async function ({ deployments, network, getNamedAccounts }: HardhatRuntimeEnvironment) {
+  if (!["mainnet", "hardhat"].includes(network.name)) {
+    throw new Error("Invalid network for mainnet deployment");
+  }
+
+  const { deployer } = await getNamedAccounts();
+
+  // await deployments.deploy("RiverV1_Implementation_1_0_1", {
```

Comment:
why is this block commented ? it seems this contract was deployed by looking at the generated artifacts

Reply:
Uncommented. Thanks